src/components/RepoList.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div id="repoList">
+    <h1>{{ userName }}</h1>
+    <div id="totalCount">
+      <p>{{ totalRepoCount }} repositories</p>
+      <p>|</p>
+      <p>{{ totalRepoStarCount }} stars</p>
+    </div>
+    <hr>
+    <p v-if="isEmptyRepoList">
+      레포지토리가 존재하지 않습니다.
+    </p>
+    <repo-list-item
+      v-else
+      v-for="(repoListItem,idx) in repoList"
+      :key="idx"
+      :repo-name="repoListItem.name"
+      :repo-star="repoListItem.stargazers_count"
+    />
+  </div>
+</template>
+
+<script>
+import RepoListItem from './RepoListItem.vue';
+
+export default {
+  name: 'RepoList',
+  components: {
+    RepoListItem,
+  },
+  props: {
+    userName: {
+      type: String,
+      default: '',
+    },
+  },
+  data() {
+    return {
+      repoList: [],
+    };
+  },
+  computed: {
+    isEmptyRepoList() {
+      return !this.repoList;
+    },
+    totalRepoCount() {
+      return this.repoList.length;
+    },
+    totalRepoStarCount() {
+      return this.repoList
+        .map((element) => element.stargazers_count, [])
+        .reduce((x, y) => x + y, 0);
+    },
```

Comment:
이런식으로 간소화 할수 있을거 같습니다!

```suggestion
    totalRepoStarCount() {
      return this.repoList
        .reduce((x, y) => x + y.stargazers_count, 0);
    },
```

Reply:
```
return this.repoList
  .reduce((accumulator, currentValue) => accumulator + currentValue.stargazers_count, 0);
```
이런식으로 수정하도록 하겠습니다! 감사합니다 ㅎㅎ